server/app/admin.py

```diff
@@ -1,32 +1,85 @@
 from django.contrib import admin
-from .models import Users, Profiles, Tasks, Bids, Payments
+from .models import Tasks, Bids, Payments
+from django.contrib.auth.admin import UserAdmin
+
+from .models import Users
 
 
-# Users Table Interface
 @admin.register(Users)
-class UsersAdmin(admin.ModelAdmin):
-    list_display = ("user_id", "email", "mobile", "created_at",
-                    "updated_at", "last_login", "status", "user_role")
-    list_filter = ("user_role", "status")
+class UserAdminConfig(UserAdmin):
+    model = Users
+    list_display = (
+        "username",
+        "email",
```

Comment:
It could be better to change the field name `email` to `account`  since users can create account with either their email or mobile.
<img width="302" alt="image" src="https://github.com/user-attachments/assets/442d9080-8ec5-47e5-b5e1-01006f33cd1e">

Didn't see the `avatar_url` and `bio` field

Reply:
sorry for now we are unable to do that cuz it will be too complicated. I think we need to change the design a little bit, will sign in or sign up with email and password.